Gemfile.lock

```diff
@@ -216,7 +216,7 @@ GEM
       thor (~> 0.14)
     formatador (0.2.5)
     foundation_emails (2.2.0.0)
-    geocoder (1.3.7)
+    geocoder (1.4.0)
```

Comment:
geocoder is a dependency of ahoy_matey, so it will automatically get updated even if ahoy_matey is the only gem that is updated via `bundle update [name of gem]`. We already use bummr on this project, but in this case, attempting to update all gems resulted in an error, and I didn't feel like waiting for bummr to run the bisect and find the culprit. So, I decided to update ahoy by itself to start working on the metrics feature, and saw that it caused specs to fail, and figured it was probably the culprit.